components/selection/selection-observer-mixin.js

```diff
@@ -16,7 +16,12 @@ export const SelectionObserverMixin = superclass => class extends superclass {
 			 * @type {object}
 			 */
 			selectionInfo: { type: Object },
-			_provider: { type: Object, attribute: false }
+			/**
+			 * HtmlElement of the `SelectionMixin` component this component wants to observe (alternative to selectionFor)
+			 * @type {string}
+			 */
+			selectionProvider: { attribute: false },
```

Comment:
Do we need both `selectionProvider` and `_provider`? A bunch of stuff uses `_provider`, so eliminating it becomes a little more involved - while other selection components reference it, it's named as a private property since it's internal to the selection components, and not expected to be referenced by anything else.

Reply:
It should be possible to only use `_provider`, but then we'd need to wire up a setter on it to handle unsubscribing/resubscribing etc. I'll take a stab at it.